main.cc

```diff
@@ -1312,7 +1312,11 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             }
 
             netw::messaging_service::scheduling_config scfg;
-            scfg.statement_tenants = { {dbcfg.statement_scheduling_group, "$user"}, {default_scheduling_group(), "$system"} };
+            scfg.statement_tenants = {
+                    {dbcfg.statement_scheduling_group, "$user"},
+                    {default_scheduling_group(), "$system"},
+                    {dbcfg.streaming_scheduling_group, "$maintenance"}
```

Comment:
What happens on upgrade?

I expect it should just work - an old server node will not understand $maintenance and direct it to the statement group.

Reply:
Yes, an old server will just fall-back to the default tenant. Which is the pre-patch behaviour.